src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -24,11 +49,152 @@ export const ChooseServicePage = () => {
 
   checkDelegationAccess();
 
+  // Temporary hardcoding of filter options
+  const filterOptions = [
+    { label: 'Påfunnsetaten', value: '130000000' },
+    { label: 'Testdepartementet', value: '123456789' },
+    { label: 'Narnia', value: '777777777' },
+    { label: 'Brannvesenet', value: '110110110' },
+    { label: 'Økern Portal', value: '904111111' },
+    { label: 'Digitaliseringsdirektoratet', value: '991825827' },
+    { label: 'Brønnøysundregistrene', value: '974760673' },
+  ];
+
+  const unCheckFilter = (filter: string) => {
+    setFilters((prev) => prev.filter((f) => f !== filter));
+    setCurrentPage(1);
+  };
+
+  const getFilterLabel = (value: string) => {
+    for (const option of filterOptions) {
+      if (option.value === value) {
+        return option.label;
+      }
+    }
+    return '';
+  };
+
+  const filterChips = () => (
+    <Chip.Group
+      size='small'
+      className={classes.filterChips}
+    >
+      {filters.map((filterValue) => (
+        <Chip.Removable
+          key={filterValue}
+          aria-label={t('common.remove') + ' ' + getFilterLabel(filterValue)}
+          onClick={() => {
+            unCheckFilter(filterValue);
+          }}
+        >
+          {getFilterLabel(filterValue)}
+        </Chip.Removable>
+      ))}
+    </Chip.Group>
+  );
+
+  const serviceResouces = resources?.map((r: ServiceResource, index: any) => (
```

Comment:
Don't use any in this context
```suggestion
  const serviceResouces = resources?.map((r: ServiceResource, index: number) => (
```